06-building-a-program-start-to-finish/writing-tests/output/test.js

```diff
@@ -0,0 +1,136 @@
+const invertedIndex = require('../src/inverted-index.js');
+const books = require('./books');
+
+describe('Inverted index Suite', () => {
+   // Create an instance of the Index class
+  const newIndex = new invertedIndex();
+  const sampleSentence = 'As &you can see here, you, have defined the function';
+  beforeAll(() => {
+    newIndex.createIndex('books', books);
+  });
+
+  describe('Class Inverted Index', () => {
+    it('should be a class', () => {
+      expect(newIndex instanceof Object).toBe(true);
+      expect(newIndex instanceof invertedIndex).toBe(true);
+      expect(typeof newIndex).toBe('object');
+    });
+  });
+
+  describe('Tokenize String', () => {
```

Comment:
Your specs are repeated. If you're testing for the same scenario, you might as well combine them or explicitly define each scenario in your specs.